plugins/circleci/api-report.md

```diff
@@ -49,33 +51,28 @@ export const circleCIApiRef: ApiRef<CircleCIApi>;
 export const circleCIBuildRouteRef: RouteRef<undefined>;
 
 // @public (undocumented)
-const circleCIPlugin: BackstagePlugin<{}, {}>;
-
+const circleCIPlugin: BackstagePlugin<    {}, {}>;
```

Comment:
There's some weird stuff going on with them here...

Reply:
Haha

Reply:
Maybe we just run prettier on these `api-reports.md` after the build. They're currently excluded for some reason right now, guessing maybe that we didn't want the warnings on them at build time.